### ISSUE TITLE
Authorization error while refreshing opentelekomcloud_cts_tracker_v1

### ISSUE THREAD

**Issue by Unichron** [opentelekomcloud/terraform-provider-opentelekomcloud#1122]
# Terraform Version
1.0.0

# Affected Resource(s)
opentelekomcloud_cts_tracker_v1

# Terraform Configuration Files
```hcl
resource "opentelekomcloud_cts_tracker_v1" "this" {
  bucket_name               = <redacted>
  is_send_all_key_operation = false
  is_support_smn            = false
  operations                = []
  project_name              = var.project
  status                    = "enabled"
  topic_id                  = ""
}
```

# Debug Output/Panic Output

# Steps to Reproduce
1. `terraform apply` to create the resource
2. `terraform apply` again

# Expected Behavior
Noop

# Actual Behavior
Error while refreshing the state of the tracker:
```
╷
│ Error: error creating cts Client: failed to authenticate:
│ error generating project client: Authentication Failed, error message: {"message":"Authorization information is wrong","request_id":"0ab707169addb6e8a3d3441b963192d3"}
│ 
│ 
│   with module.this.opentelekomcloud_cts_tracker_v1.this,
│   on ../../../modules/otc/cts/main.tf line 1, in resource "opentelekomcloud_cts_tracker_v1" "this":
│    1: resource "opentelekomcloud_cts_tracker_v1" "this" {
│ 
╵
```

**Comment by outcatcher** [comment]
Reproduced if using username/password auth. Not reproduced if using AK/SK :thinking:

**Comment by outcatcher** [comment]
Expected to be fixed by #1133 and #1134. Will require additional customer testing with release `1.24.2`.

**Comment by Unichron** [comment]
@outcatcher confirmed that it works now
